cyences_app_for_splunk/default/data/ui/views/cs_splunk_admin_reports.xml

```diff
@@ -125,4 +137,39 @@
       </table>
     </panel>
   </row>
-</form>
\ No newline at end of file
+  <row>
+    <panel>
+      <title>Disk Statistics</title>
+      <input type="dropdown" token="host">
+        <label>Host</label>
+        <choice value="*">All</choice>
+        <fieldForLabel>Host</fieldForLabel>
+        <fieldForValue>Host</fieldForValue>
+        <search base="base_search">
+          <query>| fields splunk_server | rename splunk_server as Host | dedup Host</query>
+        </search>
+        <default>*</default>
+        <initialValue>*</initialValue>
+      </input>
+      <input type="dropdown" token="disk_usage" searchWhenChanged="true">
+        <label>Disk Usage</label>
+        <choice value="&gt;0">All</choice>
+        <choice value="&gt;80">&gt; 80</choice>
+        <choice value="&gt;60">&gt;60</choice>
+        <default>&gt;0</default>
+        <initialValue>&gt;0</initialValue>
+      </input>
+      <table>
+        <search base="base_search">
+          <query>| search splunk_server=$host$ AND pct_usage$disk_usage$| rename mount_point as "Mount Point", compare_usage as "Disk Usage (GB)", capacity as "Capacity (GB)", pct_usage as "Disk Usage (%)", splunk_server as Host | search Host=$host$</query>
+        </search>
+        <option name="drilldown">none</option>
```

Comment:
I would say adding a drilldown would be helpful when some system is having the issue.
Drilldown: time chart of disk-usage over time. (may be for last 7 days would be useful.)

Reply:
Okay. I will add it.